src/ws-header/authorization.js

```diff
@@ -31,8 +26,11 @@ export class Authorization {
    * @returns {void}
    */
   changeAccessToken(accessToken) {
```

Comment:
could you just change the docs for that ? looks like you now expect a string or a function as input ?